persistence/src/commonMain/db_user/com/wire/kalium/persistence/Conversations.sq

```diff
@@ -69,6 +71,9 @@ SELECT * FROM Conversation WHERE qualified_id = ?;
 selectByGroupId:
 SELECT * FROM Conversation WHERE mls_group_id = ?;
 
+selectByGroupState:
+SELECT * FROM Conversation WHERE mls_group_state = ? AND protocol = ?;
```

Comment:
since the state is only valid for mls groups, isn't good idea to limit it protocol = mls?

Reply:
That's what I do in code. Protocol is a enum though and I couldn't figure out how use that enum in the `sq` file.

Reply:
I didn't notice that. great then! there is no clean solution for it as far as I know, if it's an enum we have to hard code it.